src/cf-layout/src/cf-layout.less

```diff
@@ -650,36 +650,35 @@
     });
   }
 }
-//
-// Hero
-//
+
 
 @hero-desktop-height: 285px;
 @hero-image-height: 195px;
-@desktop-min: 801px;
+@hero-bg: #f7f8f9;
+@hero-overlay-bg: #5a5d61;
+@hero-overlay-text: white;
 
 .hero {
-    background-color: @block__bg;
+    background-color: @hero-bg;
 
-    &_wrapper {
-        box-sizing: border-box;
-        padding: unit(@grid_gutter-width / @base-font-size-px, em)
-                 unit(15px / @base-font-size-px, em);
+    .respond-to-min( @bp-sm-min, {
+        display: table;
```

Comment:
L746 is necessary, but not sufficient. The things that are `display: table-cell;` apparently need their immediate parent to be `display: table;`.